src/components/data-management/ProjectsList.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Space } from 'antd';
+
+import ProjectCard from './ProjectCard';
+
+const ProjectsListContainer = (props) => {
+  const { height, filter } = props;
+
+  const projects = useSelector((state) => state.projects);
+
+  return (
+    <Space direction='vertical' style={{ width: '100%', height: height - 90 }}>
```

Comment:
I just checked what happens if we don't define height and guess -- nothing. The component looks the same. So you can remove all these height passings happening in this component and also from the other `ProjectsListContainer` component from from index.jsx when you call it and this should remove some extra unneeded code.

Reply:
The height here is added to constraint the height of the Space element and force it to show a scroll bar. If we remove height, when you add more projects, the scrollbar will not appear as the element's height expands to accommodate the projects.

Reply:
This is addressed in commit dd8faee. The value is changed to 130. The value comes from adding padding, button size and gap between buttons in the UI. More details can be seen in `ProjectsList` line 8-16